src/tests/log-file/out/log-file.txt

```diff
@@ -4,7 +4,7 @@ DEBUG	Build date: %s
 DEBUG	Go version: %s
 DEBUG	Os/Arch:    %s
 DEBUG	Running command [%s --log-file ./log-file.txt]
-DEBUG	Parsed options: &options.Options{Verbose:false, VerboseApi:false, LogFilePath:"./log-file.txt", ApiHost:"", ApiToken:"", workingDirectory:"%s", projectDirectory:"", metadataDirectory:""}
+DEBUG	Parsed options: %s
```

Comment:
Vypisovala sa tu adresa pointra na `*Viper`, tak som to nahradil JSON vypisom, ... tu to netreba testovat, ma to vlastny test `TestDumpOptions`.